helper/helper_test.go

```diff
@@ -49,6 +49,30 @@ func TestArmoredTextMessageEncryption(t *testing.T) {
 	assert.Exactly(t, plaintext, decrypted)
 }
 
+func TestRawMessageArmoredEncryption(t *testing.T) {
+	plaintext := "Secret message"
+
+	armored, err := EncryptRawMessageArmored(readTestFile("keyring_publicKey", false), []byte(plaintext))
+
+	if err != nil {
+		t.Fatal("Expected no error when encrypting, got:", err)
+	}
+
+	assert.Exactly(t, true, crypto.IsPGPMessage(armored))
+
+	decrypted, err := DecryptMessageArmored(
```

Comment:
Please edit `DecryptMessageArmored`, (like the encrypt function) to have a common internal function with two wrappers:
- `DecryptMessageArmored` with the same signature 
- `DecryptBinaryMessageArmored` returning `[]byte`.

If we expect bytes it makes more sense not to cast the data to a string